### PR TITLE
upadated readme

### PR DESCRIPTION
**Pull request by dylanjhall**
Daniel please approve my changes

### CODE REVIEW

**Review by HastyHD — COMMENTED** [review]
what is this ballocks!

**Comment by dylanjhall** [comment]
> what is this ballocks!

you are always picking on me !!!!!

**Comment by dylanjhall** [comment]
@HastyHD Please approve now!!!!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # GamingStats
-Smash bros results for 2022.  
+
+Smash bros results for 2022.
+
+## How to run
```
Comment (HastyHD) [inline README.md:5]: this heading could improve

**Review by HastyHD — CHANGES_REQUESTED** [review]
hmm i see something

**Review by HastyHD — APPROVED** [review]

**Pull request merged**